chicken-fries/.gitignore

```diff
@@ -0,0 +1,3 @@
+.DS_Store
+.idea
+mode_modules
```

Comment:
There's a typo here that. This is probably what's causing node_modules to sneak through the ignore settings.

Reply:
@markmichon this is strange, in my feature branch is shows node_modules and it does even show the chicken-fries folder anymore
<img width="1440" alt="screen shot 2018-09-15 at 1 59 55 pm" src="https://user-images.githubusercontent.com/3230409/45589198-c7962300-b8ef-11e8-904d-06e4b452f834.png">